guides/introspection.md

```diff
@@ -91,6 +91,43 @@ Getting the name of the fields for a named type:
 Note that you may have to nest several depths of `type`/`ofType`, as
 type information includes any wrapping layers of [List](https://spec.graphql.org/October2021/#sec-List) and/or [NonNull](https://spec.graphql.org/October2021/#sec-Non-Null).
 
+## Configuration
+
+The introspection system can be configured through your application config:
+
+```elixir
+config :absinthe,
+  include_deprecated: false  # Defaults to false, set to true for backward compatibility
+```
+
+This configuration affects the default behavior of the `includeDeprecated` argument in introspection queries. When set to `false` (the default), deprecated fields and values will not be included in introspection results unless explicitly requested via the `includeDeprecated` argument. When set to `true`, deprecated fields and values will be included by default, matching the behavior of previous versions of Absinthe.
+
```

Comment:
@filipemonteiroth can we add a sentence here indicating that setting this to true will also break compatibility?

Reply:
Perfect ty